updateMe.bat

```diff
@@ -0,0 +1,5 @@
+::Update the https://github.com/sfilipi/machinelearning-1 from https://github.com/dotnet/machinelearning
+git fetch upstream
```

Comment:
please don't checkin this file :)

Reply:
Thanks. 

---
In reply to: [198271533](https://github.com/dotnet/machinelearning/pull/383#discussion_r198271533) [](ancestors = 198271533)